exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers(first[0, 2]); // complete this statement
```

Comment:
For this function, you need to get the first item in the array which you can do this with the [0] index, like so:
```
  return arr[0]; // complete this statement
```